azure/rs_azure_sql/README.md

```diff
@@ -0,0 +1,372 @@
+- "dtu"
+- "databaseDtuMin"
+- "databaseDtuMax"
+- "storageMB"
+
+## auditing_policy
+#### Supported Fields
+| Field Name | Required? | Description |
+|------------|-----------|-------------|
+|name|Yes|The name of the sql server.|
+|resource_group|Yes|Name of resource group in which to launch the Deployment|
+|location|Yes|Datacenter to launch in|
+|server_name|Yes|Server the database is on|
+|database_name|Yes|database to configure auditing-policy on|
+|properties|Yes|Hash of Deployment properties (https://docs.microsoft.com/en-us/rest/api/sql/blob%20auditing%20policies)|
+
+#### Supported Actions
+
+| Action | API Implementation | Support Level |
+|--------------|:----:|:-------------:|
+| create&update | [Create Or Update](https://docs.microsoft.com/en-us/rest/api/sql/blob%20auditing%20policies#Databases_CreateOrUpdateBlobAuditingPolicy) | Supported |
+| get | [Get](https://docs.microsoft.com/en-us/rest/api/sql/blob%20auditing%20policies#Databases_GetBlobAuditingPolicy)| Supported |
+
+#### Supported Outputs
+- "id"
+- "name"
+- "type"
+- "location"
+- "kind"
+- "state"
+- "storageEndpoint"
+- "storageAccountAccessKey"
+- "retentionDays"
+- "storageAccountSubscriptionId"
+- "isStorageSecondaryKeyInUse"
+- "auditActionsAndGroups"
+
+## security_policy
+#### Supported Fields
+| Field Name | Required? | Description |
+|------------|-----------|-------------|
+|name|Yes|The name of the sql server.|
+|resource_group|Yes|Name of resource group in which to launch the Deployment|
+|location|Yes|Datacenter to launch in|
+|server_name|Yes|Server the database is on|
+|database_name|Yes|database to security_policy on|
+|properties|Yes|Hash of Deployment properties (https://docs.microsoft.com/en-us/rest/api/sql/database%20security%20policies)|
+
+#### Supported Actions
+
+| Action | API Implementation | Support Level |
+|--------------|:----:|:-------------:|
+| create&update | [Create Or Update](https://docs.microsoft.com/en-us/rest/api/sql/database%20security%20policies#Databases_CreateOrUpdateThreatDetectionPolicy) | Supported |
+| get | [Get](https://docs.microsoft.com/en-us/rest/api/sql/database%20security%20policies#Databases_GetThreatDetectionPolicy)| Supported |
+
+#### Supported Outputs
+- "id"
+- "name"
+- "type"
+- "location"
+- "kind"
+- "state"
+- "emailAccountAdmins"
+- "emailAddresses"
+- "disabledAlerts"
+- "retentionDays"
+- "storageAccountAccessKey"
+- "storageEndpoint"
+- "useServerDefault"
+
+## Implementation Notes
+- The Azure SQL Database Plugin makes no attempt to support non-Azure resources. (i.e. Allow the passing the RightScale or other resources as arguments to an SQL resource.) 
+
+ 
+Full list of possible actions can be found on the [Azure SQL Database API Documentation](https://docs.microsoft.com/en-us/rest/api/sql/)
+## Examples
+Please review [sql_test_cat.rb](./sql_test_cat.rb) for a basic example implementation.
+	
+## Known Issues / Limitations
+- Currently only supports CRUD and instance register/deregister functions.
```

Comment:
this looks to be leftover from ELB.  perhaps add a note about how Failover Groups is untested?